edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/KubeUtils.cs

```diff
@@ -0,0 +1,255 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using CoreConstants = Microsoft.Azure.Devices.Edge.Agent.Core.Constants;
+
+    public static class KubeUtils
+    {
+        const string Alphabet = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+        const string Numeric = "0123456789";
+        const string AllowedCharsLabelValues = "-._";
+        const string AllowedCharsDns = "-";
+        const string AllowedCharsGeneric = "-.";
+        const int MaxK8SValueLength = 253;
+        const int MaxDnsNameLength = 63;
+        const int MaxLabelValueLength = 63;
+        static readonly HashSet<char> AlphaHashSet;
+        static readonly HashSet<char> AlphaNumericHashSet;
+        static readonly HashSet<char> AllowedLabelsHashSet;
+        static readonly HashSet<char> AllowedDnsHashSet;
+        static readonly HashSet<char> AllowedGenericHashSet;
+
+        static bool IsAlpha(char ch) => AlphaHashSet.Contains(ch);
+
+        static bool IsAlphaNumeric(char ch) => AlphaNumericHashSet.Contains(ch);
+
+        static KubeUtils()
+        {
+            AlphaHashSet = new HashSet<char>();
+            foreach (char c in Alphabet)
+            {
+                AlphaHashSet.Add(c);
+            }
+            AlphaNumericHashSet = new HashSet<char>(AlphaHashSet);
```

Comment:
I think instead of static ctor you can just initialize fields in-place like this:
```
static readonly HashSet<char> AlphaNumericHashSet = new HashSet<char>((Alphabet+Numeric).ToCharArray());
```
IMO it makes code more declarative and clear

Reply:
Nice.  Will do.